app/blog/article/[slug]/page.js

```diff
@@ -36,8 +44,8 @@ export default async function Page({ params }) {
 
   const content = await fetchArticle(slug);
 
-  if (!content) {
-    redirect('/blog');
+  if (!content || content?.statusCode === 404) {
```

Comment:
is it status or statusCode?

Reply:
`statusCode`.. this is on the body of the request from the `nest` `NotFoundException` response, not the status code of the response, which we can't access here because we swallow it in the `unAuthFetch` method. 😕